apps/domestic-abuse/preview/base/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../../base    # This is apps/base instead of apps/domestic-abuse/base as preview doesn't install everything
+  - ../../identity/identity.yaml
+  - ../../../ccd/identity/identity.yaml
+  - ../../../xui/identity/identity.yaml
+namespace: domestic-abuse
+patchesStrategicMerge:
+  - ../../identity/aat.yaml
+  - ../../../ccd/identity/aat.yaml
+  - ../../../xui/identity/aat.yaml
```

Comment:
xui and ccd in this file?

Reply:
Hello,

 This is for preview setup. In preview we need to include ccd and  XUI setup so that we can test the changes. Its similar with private-law.